### PR TITLE
Update cwl-vs-wdl.md

### PR DESCRIPTION
**Pull request by aofarrel**
Based on comments by Julian and my own general distaste with the document.

### CODE REVIEW

File: _documentation_/cwl-vs-wdl.md [hunk 1]
```diff
@@ -1,45 +1,71 @@
...
+
+Be aware that the original CWL gives different output for null_model_file depending on whether the Seven Bridges backend selected at workspace creation time is AWS (default) or Google. To clarify:
+
+|                    	| WDL, local 	| WDL, Terra 	| CWL, 7B via AWS 	| CWL, 7B via Google 	|
+|--------------------	|------------	|------------	|-----------------	|--------------------	|
+| WDL, local         	| pass       	| pass       	| fail            	| pass               	|
+| WDL, Terra         	| pass       	| pass       	| fail            	| pass               	|
+| CWL, 7B via AWS    	| fail       	| fail       	| pass            	| fail               	|
+| CWL, 7B via Google 	| pass       	| pass       	| fail            	| pass               	|
```
Comment (juklucas) [inline _documentation_/cwl-vs-wdl.md:39]: You may want to switch from pass/fail to match/not match. You might also want to explain what is failing and how (MD5).

**Review by juklucas — APPROVED** [review]
LGTM. 

I guess I wished we had thought to change the localization from a copy to a move command. That way we wouldn't have to talk about "twice localized" so much.

That being said, I'm still curious that soft linking didn't work. We have used it in HPRC and it works as long as you call the file name (instead of the file path).

File: _documentation_/cwl-vs-wdl.md (hunk 1 above)
Comment (aofarrel) [reply _documentation_/cwl-vs-wdl.md:39]: This section doesn't refer to MD5 at all, so I definitely need to clarify it.

**Comment by aofarrel** [comment]
Merging now so I can rebase on the rearrangement branch.

**Pull request merged**